src/doc/data_into_visit/VTKFormat.rst

```diff
@@ -0,0 +1,549 @@
+.. _Data_Into_VisIt_VTK:
+
+The VTK file format
+===================
+
+There are primarily two types of VTK files, legacy and XML.
+We are going to focus on the legacy files, since they are a little bit simpler.
+
+* VTK files can represent a single timestep.
+
+  * If you have multiple times you can group them with a `.visit` file.
+
+* VTK files can be text or binary
+
+  * Both text and binary files have text meta data.
+  * Binary files have binary fields and coordinates imbedded in the text.
+
+* VisIt_ has conventions for additional meta data that fits within the VTK specification.
+
+The remainder of this VTK documentation consistes of a description of the file format, a simple example, and then more complex examples of the different mesh types.
+
+* For a description of a VTK file go :ref:`here<data_into_visit_vtk_structure>`.
+* For a description of the VisIt_ VTK meta data conventions go :ref:`here<data_into_visit_vtk_conventions>`.
+* For an example VTK file go :ref:`here<data_into_visit_vtk_example>`.
+* For an example of a VTK file with extra metadata go :ref:`here<data_into_visit_vtk_metadata_example>`.
+* For an example structured points file go :ref:`here<data_into_visit_vtk_struct_points>`.
+* For an example structured grid file go :ref:`here<data_into_visit_vtk_struct_grid>`.
+* For an example rectilinear grid file go :ref:`here<data_into_visit_vtk_rect_grid>`.
+* For an example polydata file go :ref:`here<data_into_visit_vtk_polydata>`.
```

Comment:
Now that I look at these 'go here' references again in the generated docs, they seem a bit unnecessary.The TOC on the left shows how the current page is sectioned. Shouldn't that be sufficient? 
For this VTK page, it is already stated just before the 'go here' list what the rest of the page will be discussing.
Keep the link to VTK official docs though.
The very short pages like BOV really don't need the 'go here' links.

Reply:
Completed.